exercises/word-count/canonical-data.json

```diff
@@ -142,6 +142,18 @@
         "multiple": 1,
         "whitespaces": 1
       }
+    },
+    {
+      "description": "alternating word separators not detected as a word",
+      "property": "countWords",
+      "input": {
+        "sentence": ",\n,one,\n ,two \n 'three'"
+      },
+      "expected": {
+        "one": 1,
+        "two": 1,
+        "three": 1
+      }
     }
   ]
-}
\ No newline at end of file
+}
```

Comment:
unrelated change.

Reply:
I changed the file here on github. It seems i adds that automatically. I tried to remove it. I can't. Should be there anyway.

Reply:
Okay, so how about a commit that adds the newline and then the commit that adds the test. at least that way, it's separate in the commit history

Reply:
added separate commit for line ending